constellations/gemini-admin/src/help/help.js

```diff
@@ -5,7 +5,7 @@ import React from "react";
 export class HelpPage extends React.Component {
     render() {
         return (
-            <div>
+            <div id="view-help">
```

Comment:
Let's try to minimize the merge conflict @DanielLiu14 is going to have in #178 so let's get rid of all the changes in help-related stuff.